app/javascript/readingList/readingList.jsx

```diff
@@ -187,17 +193,17 @@ export class ReadingList extends Component {
       ''
     );
     return (
-      <section>
-        <header className="crayons-layout flex justify-between items-center pb-0">
+      <main>
+        <header className="crayons-layout l:grid-cols-2 pb-0">
           <h1 class="crayons-title">
             {isStatusViewValid ? 'Reading list' : 'Archive'}
             {` (${itemsTotal})`}
           </h1>
-
-          <div class="flex items-center">
+          <fieldset className="grid gap-2 m:flex m:justify-end l:mb-0 mb-2 px-2 m:px-0">
```

Comment:
If you add `m:items-center` it will better display button and input (right now, it's kinda enlarged).

Reply:
Done.